packages/core/theme/components/breadcrumbs/index.js

```diff
@@ -3,7 +3,7 @@ export const theme = {
     p: 1
   },
   colors: {
-    color: "gray3",
+    color: "gray500",
```

Comment:
This is a quick fix that was missed in a previous PR. gray3 I no longer a color.